### PR TITLE
Riedl

### PR DESCRIPTION
**Pull request by jriedl**

### CODE REVIEW

File: cpp/bibelwissenschaft/extract_names.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+MARC_FILE=$1
+OUTFILE_PREFIX=$2
+
+for bibwiss_type in "WiReLex" "WiBiLex"; do
+    marc_grep $1 'if "TYP"=="'${bibwiss_type}'" extract "856u"' traditional | \
+    awk '{print $2}' | \
+    xargs -I '{}' /bin/bash -c $'
+        interval=$((1 + $RANDOM % 3)); echo "Sleeping ${interval} seconds" >2; sleep ${interval};
+        ./translate_url_multiple "$1" | \
```
Comment (mtrojan-ub) [inline cpp/bibelwissenschaft/extract_names.sh:16]: Shouldn't the extension of translate_url_multiple be .sh if it is a shell script?

File: cpp/bibelwissenschaft/add_bibwiss_authors_to_marc.cc [hunk 2]
```diff
@@ -0,0 +1,105 @@
+/** \brief Augment Bibelwissenschaften encyclopaedia MARC data with authors and references as extracted by the
+ * extract_authors_and_references script \author Johannes Riedl
```
Comment (mtrojan-ub) [inline cpp/bibelwissenschaft/add_bibwiss_authors_to_marc.cc:2]: mssing newline before \author

File: cpp/bibelwissenschaft/convert_bibwiss_to_marc.cc [hunk 3]
```diff
@@ -206,18 +237,27 @@ void ConvertArticles(DbConnection * const db_connection, const DbFieldToMARCMapp
         db_connection->queryOrDie(bibwiss_query);
         DbResultSet result_set(db_connection->getLastResultSet());
         while (const auto row = result_set.getNextRow()) {
+            // Apparently the entry does not exist in the web version no pdf is was generated
```
Comment (mtrojan-ub) [inline cpp/bibelwissenschaft/convert_bibwiss_to_marc.cc:240]: The words in this comment seem to be mixed up

File: cpp/bibelwissenschaft/extract_names.sh (hunk 1 above)
Comment (jriedl) [reply cpp/bibelwissenschaft/extract_names.sh:16]: As explained.

**Pull request merged**